dist/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To do list</title>
+  <script defer src="to-do-list.js"></script></head>
+  <body>
+    <div id="to-do-list">
+      <ul class="to-do-list">
+        <li
+          >Today's To Do
+          <img id="reload" src="./icons8-refresh-30.svg" alt="reload img"
+        /></li>
+        <li
+          ><input
+            id="input"
+            type="text"
+            name="add to list"
+            placeholder="Add to your list ..."
+            required /><button type="submit" class="input-btn"
+            ><ion-icon
+              class="list-icon"
+              name="return-down-back-outline"
+            ></ion-icon></button
+        ></li>
+        <li
+          ><button type="button" class="clear"> Clear all completed</button></li
+        >
+      </ul>
+    </div>
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"
+    ></script>
+  </body>
+</html>
```

Comment:
Kindly try to use HTML semantic tags like header, footer for HTML best practice, and to make your code more meaningful.